content/references/rippled-api/api-conventions/serialization.md

```diff
@@ -0,0 +1,285 @@
+| [UInt32][]    | 2         | No               | A 32-bit unsigned integer. The `Flags` and `Sequence` fields on all transactions are examples of this type. |
+
+In addition to all of the above field types, the following types may appear in other contexts, such as [ledger objects](ledger-object-types.html) and [transaction metadata](transaction-metadata.html):
+
+| Type Name   | Type Code | Variable-Length? | Description                     |
+|:------------|:----------|:-----------------|:--------------------------------|
+| Transaction | 10001     | No               | A "high-level" type containing an entire [transaction](transaction-formats.html). |
+| LedgerEntry | 10002     | No               | A "high-level" type containing an entire [ledger object](ledger-object-types.html). |
+| Validation  | 10003     | No               | A "high-level" type used in peer-to-peer communications to represent a validation vote in the [consensus process](consensus.html). |
+| Metadata    | 10004     | No               | A "high-level" type containing [metadata for one transaction](transaction-metadata.html). |
+| [UInt64][]  | 3         | No               | A 64-bit unsigned integer. This type does not appear in transaction instructions, but several ledger objects use fields of this type. |
+| Vector256   | 19        | Yes              | This type does not appear in transaction instructions, but the [Amendments ledger object](amendments-object.html)'s `Amendments` field uses this to represent which [amendments](amendments.html) are currently enabled. |
+
+
+### AccountID Fields
+[AccountID]: #accountid-fields
+
+Fields of this type contain the 160-bit identifier for an XRP Ledger [account](accounts.html). In JSON, these fields are represented as base58 XRP Ledger "addresses", with additional checksum data so that typos are unlikely to result in valid addresses. (This encoding, sometimes called "Base58Check", prevents accidentally sending money to the wrong address.) The binary format for these fields does not contain any checksum data. (However, since the binary format is used mostly for signed transactions, a typo or other error in transcribing a signed transaction would invalidate the signature, preventing it from sending money.)
+
+AccountIDs that appear as stand-alone fields (such as `Account` and `Destination`) are [variable-length encoded](#variable-length-encoding) despite being a fixed 160 bits in length. As a result, the length indicator for these fields is always the byte `0x14`. AccountIDs that appear as children of special fields ([Amount `issuer`][Amount] and [PathSet `account`][PathSet]) are _not_ variable-length encoded.
+
+
+### Amount Fields
+[Amount]: #amount-fields
+
+The "Amount" type is a special field type that represents an amount of currency, either XRP or an issued currency. This type consists of two sub-types:
+
```

Comment:
Just an FYI that the formatting of the following bolded **XRP** and **Issued Currencies** bullets is a little off. The sentences following the bullets aren't displaying on the next line, but display immediately after the bullet text.